examples/ola-recorder.cpp

```diff
@@ -55,15 +55,19 @@ DEFINE_default_bool(verify_playback, true,
 DEFINE_s_string(universes, u, "",
                 "A comma separated list of universes to record");
 DEFINE_s_uint32(delay, d, 0, "The delay in ms between successive iterations.");
-DEFINE_uint32(duration, 0, "The length of time (seconds) to run for.");
+DEFINE_uint32(duration, 0, "Total playback time (milliseconds); the program "
+                           "will close after this time has elapsed. This "
+                           "option overrides the iteration option.");
 // 0 means infinite looping
 DEFINE_s_uint32(iterations, i, 1,
-                "The number of times to repeat the show, 0 means unlimited.");
+                "The number of times to repeat the show, 0 means unlimited. "
+                "The duration option overrides this option.");
```

Comment:
I know we don't always verify, but will we get the log message if it does verify and the two choices conflict?

Reply:
Adding a warning message for this case